<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellerTorrentPluginConfigurationDialog</class>
 <widget class="QDialog" name="SellerTorrentPluginConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>177</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="sellerTorrentPluginConfigurationLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>8</y>
     <width>253</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Seller</string>
   </property>
  </widget>
  <widget class="QLabel" name="minPriceLabel">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>52</y>
     <width>85</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>minPrice (mBTC):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="minPriceLineEdit">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>48</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="minLockLabel">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>78</y>
     <width>139</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>minLock (hours::minutes):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="minFeeLineEdit">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>102</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="minFeeLabel">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>106</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>minFeet/KB (mBTC):</string>
   </property>
  </widget>
  <widget class="QLabel" name="maxmaxContractConfirmationDelayLabel">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>132</y>
     <width>203</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>max Confirmation (hours::minutes):</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="maxContractConfirmationDelayTimeEdit">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>128</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="minLockTimeEdit">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>72</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SellerTorrentPluginConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SellerTorrentPluginConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
